tests/foreman/ui/test_webhook.py

```diff
@@ -69,6 +73,9 @@ def test_positive_end_to_end(session):
         assert values['general']['enabled'] is False
         assert values['credentials']['capsule_auth'] is True
         assert values['credentials']['verify_ssl'] is False
+        assert values['credentials']['user'] == username
+        result = target_sat.execute('echo "Webhook.last.password" | foreman-rake console')
+        assert password in result.stdout
```

Comment:
As per BZ, perhaps you might check `Webhook.last.password` again after editing the webhook.

Reply:
In the UI for webhooks, reading and editing is the same action except editing actually submits changes. It still takes you to the edit screen to read all the values, so it would be redundant to do the check again.

Reply:
Okay, will resolve this thread